src/ildasm/dis.cpp

```diff
@@ -1646,7 +1646,7 @@ BOOL Disassemble(IMDInternalImport *pImport, BYTE *ILHeader, void *GUICookie, md
 
             case ShortInlineBrTarget:
             {
-                char offset = (char) pCode[PC];
+                signed char offset = (signed char) pCode[PC];
```

Comment:
I think using `-fisnged-char` may solve the problem.
However I prefer modifying C/C++ source code to `signed char` if we need signed `char`, because C/C++ doesn't define char is sign or unsigned.
And compiler suggests using `signed char` and `unsigned char` when sign of char matters.

Source: https://gcc.gnu.org/onlinedocs/gcc/C-Dialect-Options.html
GCC also suggests that
"-funsigned-char
Let the type char be unsigned, like unsigned char.
Each kind of machine has a default for what char should be. It is either like unsigned char by default or like signed char by default.

Ideally, a portable program should always use signed char or unsigned char when it depends on the signedness of an object. But many programs have been written to use plain char and expect it to be signed, or expect it to be unsigned, depending on the machines they were written for. This option, and its inverse, let you make such a program work with the opposite default.

The type char is always a distinct type from each of signed char or unsigned char, even though its behavior is always just like one of those two. "